packages/nodes-base/nodes/SyncroMSP/v1/actions/contact/getAll/description.ts

```diff
@@ -0,0 +1,49 @@
+import {
+	ContactProperties,
+} from '../../Interfaces';
+
+export const contactGetAllDescription: ContactProperties = [
+	{
```

Comment:
We are missing the `Limit` field here, when `Return all` is false.

Reply:
limit/perPage parameter is not available for contacts